woocommerce-gateway-stripe.php

```diff
@@ -453,6 +453,38 @@ public static function log( $message ) {
 			error_log( $message );
 		}
 	}
+	/**
+	 * Gets saved tokens from API if they don't already exist in WooCommerce.
+	 * @param array $tokens
+	 * @return array
+	 */
+	public function woocommerce_get_customer_payment_tokens( $tokens, $customer_id, $gateway_id ) {
```

Comment:
@gedex interesting! on my localhost the function was missing. Inspecting locally again.